google-cloud-bigtable-emulator/pom.xml

```diff
@@ -128,6 +128,12 @@
 
   <dependencyManagement>
     <dependencies>
+      <!-- override protobuf-java version -->
+      <dependency>
+        <groupId>com.google.protobuf</groupId>
+        <artifactId>protobuf-java</artifactId>
+        <version>${protobuf-java-test.version}</version>
+      </dependency>
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-bigtable-deps-bom</artifactId>
```

Comment:
same comment - I think adding this change just to the deps-bom should be sufficient as it should pull the version number from there.

Reply:
Sure, that makes sense. I will update the BOM when a release is required